setup.cfg

```diff
@@ -0,0 +1,2 @@
+[tool:pytest]
+addopts =-x -v -rw --durations=10 --cov=spyder --cov-report=term-missing
```

Comment:
I find the options annoying because they produce too much output, making it hard for me to see the information I am usually interested in (do all tests pass, and if not, which tests fail). But if this annoyance isn't shared, then by all means leave things as they are and I'll find a way to override the options if I want.